mapie/quantile_regression.py

```diff
@@ -0,0 +1,425 @@
+from __future__ import annotations
+from typing import Optional, Tuple, Union, cast, Iterable, List
+
+import numpy as np
+from sklearn.base import RegressorMixin, clone
+from sklearn.linear_model import QuantileRegressor
+from sklearn.pipeline import Pipeline
+from sklearn.utils.validation import (
+    indexable,
+    check_is_fitted,
+    _num_samples,
+    _check_y,
+)
+from typing_extensions import TypedDict
+
+from ._typing import ArrayLike, NDArray
+from .utils import (
+    check_alpha_and_n_samples,
+    check_n_features_in,
+    check_null_weight,
+    fit_estimator,
+    check_lower_upper_bounds,
+)
+from ._compatibility import np_quantile
+from .regression import MapieRegressor
+
+
+class MapieQuantileRegressor(MapieRegressor):
+    """
+    This class implements the conformalized quantile regression strategy
+    to make conformal predictions. The only valid ``method`` is "quantile"
+    and the only valid default ``cv`` is "split".
+    References
+    ----------
+    Yaniv Romano, Evan Patterson and Emmanuel J. Candès.
+    "Conformalized Quantile Regression"
+    https://proceedings.neurips.cc/paper/2019/hash/5103c3584b063c431bd1268e9b5e76fb-Abstract.html
```

Comment:
I would remove the link and just include the reference.

Reply:
done